docs/user-guide/cli-installplugins.md

```diff
@@ -2,7 +2,11 @@
 
 Use commands in the `plugins` command group to install and manage Zowe&trade; CLI plug-ins.
 
-**Important!** Plug-ins can gain control of your CLI application legitimately during the execution of commands. Install third-party plug-ins at your own risk. We make no warranties regarding the use of third-party plug-ins.
+:::info Important
+
+Plug-ins can gain control of your CLI application legitimately during the execution of commands. Install third-party plug-ins at your own risk. We make no warranties regarding the use of third-party plug-ins.
```

Comment:
Rather than "your CLI application", should we say "Plug-ins can gain control of Zowe CLI...". 

I would also consider removing the last sentence ("We make no warranties...")